build.gradle.kts

```diff
@@ -9,295 +10,317 @@ import org.jetbrains.intellij.tasks.*
 import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
 
 plugins {
-    checkstyle
-    `java-library`
-    scala
-    groovy
+  checkstyle
+  `java-library`
+  scala
+  groovy
 
-    alias(libs.plugins.taskTree)
-    alias(libs.plugins.versions)
-    alias(libs.plugins.versionCatalogUpdate)
+  alias(libs.plugins.taskTree)
+  alias(libs.plugins.versions)
+  alias(libs.plugins.versionCatalogUpdate)
 
-    alias(libs.plugins.grgit) apply false
-    alias(libs.plugins.jetbrains.grammarkit) apply false
+  alias(libs.plugins.grgit) apply false
+  alias(libs.plugins.jetbrains.grammarkit) apply false
 }
 
 if (JavaVersion.current() != JavaVersion.VERSION_11) {
-    throw GradleException("Project must be built with Java version 11")
+  throw GradleException("Project must be built with Java version 11")
 }
 
 val javaMajorVersion by extra(JavaVersion.VERSION_11)
 
 val intellijSnapshotsUrl by extra("https://www.jetbrains.com/intellij-repository/snapshots/")
 
 // See https://www.jetbrains.com/intellij-repository/releases/ -> Ctrl+F .idea
-val intellijVersions = IntellijVersionHelper.getInstance()
+val intellijVersions = IntellijVersionHelper.instance
 
 val ciBranch by extra(System.getenv("CIRCLE_BRANCH"))
 val isCI by extra(System.getenv("CI") == "true")
 val jetbrainsMarketplaceToken by extra(System.getenv("JETBRAINS_MARKETPLACE_TOKEN"))
 
-// This values can't be named in the same way as their corresponding properties to avoid a name clash.
-val jvmArgsForJavaCompilation by extra((project.properties["compileJavaJvmArgs"] as String?)?.split(" "))
+// This values can't be named in the same way as their corresponding properties to avoid a name
```

Comment:
What's the line lenght limit enforced by Spotless now? I'd increase it a bit to avoid such weird line splits